tasks/managed/embargo-check/embargo-check.yaml

```diff
@@ -36,6 +39,12 @@ spec:
   steps:
     - name: check-issues
       image: quay.io/konflux-ci/release-service-utils:0f82be4be43294b6a96846d87ef7f7c0b9e34267
+      env:
+        - name: ACCESS_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: todo
```

Comment:
This should not be merged with the name and key as todo (e2e) won't pass anyways. It is also in the mocks. Waiting on Rogue to provide the key and releng to add it to update this